ui/css/oae.search.css

```diff
@@ -13,36 +13,63 @@
  * permissions and limitations under the License.
  */
 
-/* Search clip overrides */
-
-#search-clip-container .oae-clip-content {
-    padding: 15px;
-}
-
-#search-clip-container .oae-clip {
-    max-width: 550px;
-    width: 80%;
-}
 
 /* Search form */
 
 #search-form {
-    margin: 0 20px;
+    margin: 0 auto 50px auto;
+    max-width: 630px;
+    padding: 0 15px;
+}
+
+.anon #search-form {
+    margin-bottom: 15px;
 }
 
 #search-form #search-search-icon {
     margin-left: -27px;
     margin-top: 5px;
-    padding: 0px;
     opacity: 0.6;
+
+    /* The typical search icon gets chopped a bit on the left side by the button container, shift it
+       over a little bit */
+    padding: 0 0 0 1px;
```

Comment:
The search icon is too much to the right compared to the icon in the topnavigation. They should be identical